### DISCUSSION TITLE
Using Google Container Registry for custom docker image

### DISCUSSION THREAD

**Discussion by ee07dazn** [strimzi/strimzi-kafka-operator#6658] [Q&A] [answered]
Hello 
I am new to Strimzi and still figuring out what all i can/cannot do for my use-case, so please bear me on this. 

I understand Strimzi supports pushing custom images to our own container registry such  DockerHub/Quay. Does that support also extend to cloud container registry like Google Container Registry and/or Amazon Container Registry ?

Thanks

**Comment by scholzj** [comment, marked as answer]
Are you talking about building Strimzi from source code? Or about using the Kafka Connect Build and adding custom connector plugins to it?

**Reply by ee07dazn** [reply]
Thanks for looking into this. 

I am talking in regards to the latter i.e. - two approaches mentioned here - https://strimzi.io/docs/operators/latest/deploying.html#using-kafka-connect-with-plug-ins-str : 

Ideally, we would like to store the images into our Google Container registry.  

Thanks

**Reply by scholzj** [reply]
Yeah, you can just create a push secret with the credentials and then you can push into any container registry: https://strimzi.io/docs/operators/latest/full/configuring.html#output
